streams/it.m3u

```diff
@@ -681,3 +681,5 @@ https://rtmp-live-ingest-us-east-1-universe-dacast-com.akamaized.net/transmuxv1/
 http://77.83.120.135:8001/play/a11n
 #EXTINF:-1 tvg-id="Rai1.it@HD",Rai 1 HD (720p)
 http://77.83.120.135:8001/play/a11c
+#EXTINF:-1 tvg-id="radiobrianza.it" tvg-name="Radio Brianza TV", group-title="General",Radio Brianza TV

```

Comment:
Wrong ID.

https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#how-to-add-a-new-stream-link-to-a-playlists